docs/modules/traverse.md

```diff
@@ -0,0 +1,47 @@
+# traverse
+
+Traverse and transform objects by visiting every node on a recursive walk.
+
+```js
+var traverse = require('traverse');
+var obj = [ 5, 6, -3, [ 7, 8, -2, 1 ], { f : 10, g : -13 } ];
+
+traverse(obj).forEach(function (x) {
+    if (x < 0) this.update(x + 128);
+});
+
+console.dir(obj);
+```
+
+# Alternatives
+
+## neotraverse
+
+Traverse and transform objects by visiting every node on a recursive walk. This is a fork and TypeScript rewrite of [traverse](https://github.com/ljharb/js-traverse) with 0 dependencies and major improvements:
+
+- 🤌 1.38KB min+brotli
```

Comment:
I'm not sure the description needs to be this long. This info is better living in the project's own readme. We only need a short description here of how it relates to the original module